src/main/views/hearing-list.njk

```diff
@@ -0,0 +1,77 @@
+{% from "govuk/components/table/macro.njk" import govukTable as 'govukTable' %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink as 'govukBackLink' %}
+
+{% extends "template.njk" %}
+{% block pageTitle %}
+    {{ "Hearings List" | safe }}
+{% endblock %}
+
+{% block beforeContent %}
+  {{ govukBackLink({
+    text: "Back",
+    href: "/"
```

Comment:
your back link goes all the way home, it should go back a step

Reply:
should just go back

Reply:
Good spot. I've changed this so that it uses the referrer passed through via the controller.

I tried it out by redirecting the home page to here locally and seemed to work OK